Microservices/user-service/src/main/java/app/sportahub/userservice/model/user/User.java

```diff
@@ -0,0 +1,28 @@
+package app.sportahub.userservice.model.user;
+
+import app.sportahub.userservice.model.BaseEntity;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@EqualsAndHashCode(callSuper = false)
+@SuperBuilder(setterPrefix = "with")
+@Document("user")
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@Data
+public class User extends BaseEntity {
```

Comment:
would the user class not require a password ?

Reply:
The password will be managed by Keycloak and will not be handled by us in the database

Reply:
I have not setup Auth yet

Reply:
Ok perfect ! Thanks !